service/controller/v13/cloudconfig/cloud_config_test.go

```diff
@@ -3,13 +3,14 @@ package cloudconfig
 import (
 	"bytes"
 	"compress/gzip"
+	"context"
 	"encoding/base64"
 	"io"
 	"strings"
 	"testing"
 
-	"github.com/aws/aws-sdk-go/service/kms"
 	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/aws-operator/service/controller/v13/encrypter"
```

Comment:
Would be cool to have the operator import in its own separate block. Same is above.

Reply:
Sure, done.